### PR TITLE
Feature/csi 4986 remove existing pvcs from vg

### PR DESCRIPTION
**Pull request by matancarmeli7**
related PRS:

- #4
- #5

### CODE REVIEW

File: controllers/volumegroup_controller.go [hunk 1]
```diff
@@ -215,6 +219,42 @@ func makeVolumeGroupName(prefix string, volumeGroupUID string) (string, error) {
 	return fmt.Sprintf("%s-%s", prefix, volumeGroupUID), nil
 }
 
+func (r *VolumeGroupReconciler) removeVolumesFromVG(logger logr.Logger, vg *volumegroupv1.VolumeGroup) error {
+	pvcList, err := utils.GetPVCList(logger, r.Client, r.DriverConfig.DriverName)
```
Comment (ArbelNathan) [inline controllers/volumegroup_controller.go:223]: if pvclist of vg is empty, should we still do all the checks?

Comment (matancarmeli7) [reply controllers/volumegroup_controller.go:223]: if the `pvcList` is empty, so it won't get into the for loop, which checks do you mean?

Comment (ArbelNathan) [reply controllers/volumegroup_controller.go:223]: I mean the `vg.pvcList`, not the one in line 223.
the checks in line 229.
if the `vg.pvcList` is empty, should we do anything here? is there anything to remove?

Comment (matancarmeli7) [reply controllers/volumegroup_controller.go:226]: got you

Comment (matancarmeli7) [reply controllers/volumegroup_controller.go:226]: done, handled it

Comment (ArbelNathan) [reply controllers/volumegroup_controller.go:226]: Tnx

**Review by ArbelNathan — DISMISSED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**